src/adapter/edges.rs

```diff
@@ -162,6 +162,37 @@ pub(super) fn resolve_function_like_edge<'a>(
     }
 }
 
+pub(super) fn resolve_module_edge<'a>(
+    contexts: ContextIterator<'a, Vertex<'a>>,
+    edge_name: &str,
+    current_crate: &'a IndexedCrate<'a>,
+    previous_crate: Option<&'a IndexedCrate<'a>>,
+) -> ContextOutcomeIterator<'a, Vertex<'a>, VertexIterator<'a, Vertex<'a>>> {
+    match edge_name {
+        "item" => resolve_neighbors_with(contexts, move |vertex| {
+            let origin = vertex.origin;
+            let module_item = vertex.as_module().expect("vertex was not a Module");
+
+            let item_index = match origin {
+                Origin::CurrentCrate => &current_crate.inner.index,
+                Origin::PreviousCrate => {
+                    &previous_crate
+                        .expect("no previous crate provided")
+                        .inner
+                        .index
+                }
+            };
+
+            Box::new(module_item.items.iter().filter_map(move |item_id| {
+                item_index
+                    .get(item_id)
+                    .map(|item| origin.make_item_vertex(item))
```

Comment:
Throw a check for `supported_item_kind` here after resolving the item and before making the vertex, that's causing the error you're seeing in the test. Rustdoc's `Import` item type (which, confusingly, represents re-exports) isn't part of our schema at the moment so we shouldn't represent it as a vertex.

Now that I think of it, we should perhaps modify `make_item_vertex()` to call that function internally and return `Option<Vertex>` instead of `Vertex`... But that can go in a separate PR if you'd like, just to keep the scope here limited.